packages/contracts-bedrock/scripts/checks/semver-natspec/main.go

```diff
@@ -129,9 +129,14 @@ func run() error {
 				return
 			}
 
+			// Skip mock contracts
+			if strings.Contains(contractName, "_MockContract") {
```

Comment:
Not a strong opinion but could just be open to making it `mock`, case insensitive. Tradeoff in explicit vs implicit